tests/unittests/unittests.cpp

```diff
@@ -42,6 +43,18 @@ struct Foo
 };
 
 
+class Wrapper
```

Comment:
Missing `#if MOODYCAMEL_HAS_EMPLACE`. The tests won't compile on older MSVCs otherwise...

Reply:
Done.